ui-v2/electron/main.ts

```diff
@@ -0,0 +1,160 @@
+import * as fs from 'fs/promises';
+import * as path from 'path';
+
+import { app, BrowserWindow, ipcMain, clipboard, dialog, session } from 'electron';
+import type {
+  IpcMainInvokeEvent,
+  OnHeadersReceivedListenerDetails,
+  HeadersReceivedResponse,
+} from 'electron';
+
+const isDevelopment = !app.isPackaged;
+
+// Enable sandbox before app is ready
+app.enableSandbox();
```

Comment:
Interesting! Hadn't seen this option before but I wonder if it gives us some more peace of mind about rendering untrusted content?